src/aggregate/aggregate.h

```diff
@@ -95,6 +103,11 @@ typedef struct {
   /** Cursor settings */
   unsigned cursorMaxIdle;
   unsigned cursorChunkSize;
+
+  /** Profile variables */
+  clock_t initTime; // Time of start. Reset for each cursor call
+  double totalTime; // Total time. Used to accimulate cursors times
+  double parseTime; // Time for parsing the query
```

Comment:
Better to use `clock_t` values instead of opaque `double`.

Reply:
These are already results of `clock() - initTime / millisec`.
Maybe change first to `initClock` will be clearer?